opentelemetry-api/src/opentelemetry/util/_importlib_metadata.py

```diff
@@ -12,27 +12,161 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# type: ignore
+
 from sys import version_info
+from typing import Dict, Tuple, Union
+
+# This is a cache to avoid going through creating a dictionary for all entry
+# points for 3.7 every time the entry points function is called.
+result_entry_points_37 = {}
```

Comment:
can this be moved inside the `if version_info.minor == 7:` block?

Reply:
Right ,fixed.